zerver/views/messages.py

```diff
@@ -264,25 +264,28 @@ def by_pm_with(self, query, operand, maybe_negate):
 
     def by_search(self, query, operand, maybe_negate):
         # type: (Query, str, ConditionTransform) -> Query
+        rendered_operand = render_markdown(None, operand)
+        rendered_operand = re.sub('<p>|</p>|<br>', '', rendered_operand)
```

Comment:
I think you can make your code more readable by this `rendered_operand.replace('<p>', '').replace('</p>', '').replace('<br>", '')` or
```Python
for expression in ('<p>', '</p>', '<br>'):
    rendered_operand = rendered.replace(expression, '')
```